api/swagger.yaml

```diff
@@ -59,6 +59,42 @@ paths:
           description: Error
           schema:
             $ref: "#/definitions/error"
+  /import:
+    post:
+      tags:
+        - Import
+      operationId: import
+      summary: "WIP/DO NOT USE: Import a zip package"
```

Comment:
```suggestion
      summary: "Alpha: Import a zip package"
      description: "This API is still in alpha state and we do not recommend its usage outside of testing purposes"
```

Reply:
@christian-kreuzberger-dtx please check if this is acceptable